src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
-body {
-  margin: 0;
+@import "./reset";
+@import "./tools/variables";
+@import "./tools/mixin";
+
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap-size;
+  padding-top: 40px;
```

Comment:
It depends on your "transform: translate(0, -20px);", isn't it? You can create variable and use it here and in transform